app/views/catalog/_question_modal.html.erb

```diff
@@ -21,9 +21,9 @@
               <p>Has your question already been answered?</p>
               <div id="s-la-widget-850"></div>
               <div class="faq-set" style="padding-top: 10px;">
-                <p class="faq"><a href="https://faq.library.princeton.edu/rbsc/faq/26972" target="_blank">How can I access Princeton University Library special collections?</a></p>
-                <p class="faq"><a href="https://faq.library.princeton.edu/rbsc/faq/64876" target="_blank">How can I purchase copies from rare books and special collections?</a></p>
-                <p class="faq"><a href="https://faq.library.princeton.edu/rbsc/faq/56766" target="_blank">Do I need permission to quote or publish from rare books and special collections?</a></p>
+                <p class="faq"><a href="https://library.princeton.edu/special-collections/faqs/how-can-i-access-special-collections" target="_blank">How can I access Princeton University Library special collections?</a></p>
```

Comment:
Can we make special collections caps as in https://library.princeton.edu/special-collections/faqs/how-can-i-access-special-collections?